tests/unit/testMetrics.js

```diff
@@ -4,28 +4,31 @@ import Datastore from '../../src/lib/Datastore';
 import ListMetrics from '../../src/lib/ListMetrics';
 import { generateStateKey, generateKey } from '../../src/lib/schema';
 import { Logger } from 'werelogs';
-import metricResponseJSON from '../../models/metricResponse';
+import s3metricResponseJSON from '../../models/s3metricResponse';
 const logger = new Logger('UtapiTest');
 const memBackend = new MemoryBackend();
 const datastore = new Datastore();
 const resourceNames = {
     bucket: 'foo-bucket',
     accountId: 'foo-account',
+    service: 's3',
 };
 const metricLevels = {
     bucket: 'buckets',
     accountId: 'accounts',
+    service: 'service',
 };
 datastore.setClient(memBackend);
 
 // Create the metric response object for a given metric.
 function getMetricResponse(schemaKey) {
     // Use `JSON.parse` to make deep clone because `Object.assign` will
     // copy property values.
-    const response = JSON.parse(JSON.stringify(metricResponseJSON));
+    const response = JSON.parse(JSON.stringify(s3metricResponseJSON));
     const responseKeys = {
         bucket: 'bucketName',
         accountId: 'accountId',
+        service: 'serviceName',
```

Comment:
This changes the defined spec. I updated the specification docs. to reflect the same.